config/filament-jobs-monitor.php

```diff
@@ -2,5 +2,13 @@
 
 // config for Croustibat/FilamentJobsMonitor
 return [
+    'navigation_enabled' => true,
 
+    'labels' => [
+        'navigation_group' => 'Settings',
+    ],
+
+    'icons' => [
+        'navigation' => 'heroicon-o-chip',
+    ],
```

Comment:
I would provide a more readable approach for settings, something like this :

 
```suggestion
    'navigation' => [
        'enabled' => true,
        'group_labels' => 'Settings',
        'icons' => 'heroicon-o-chip',
    ],
```